InTune/InTune/ViewControllers/EditProfController.swift

```diff
@@ -8,23 +8,42 @@
 
 import UIKit
 
+/*
+ add password change + email change
+ add tag cv
+ */
+
 class EditProfController: UIViewController {
 
+    @IBOutlet private var editImageView: DesignableImageView!
+    @IBOutlet private var usernameText: UITextField!
```

Comment:
Outlets should always have type prefix.
` usernameTextField: UITextField!`
`bioTextView: UITextView!`